package/yast2-fonts.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Feb 20 13:49:43 UTC 2018 - pgajdos@suse.com
+
+- bump version
```

Comment:
well, I think you need some reasoning and also bsc or fate to pass automatic checks

Reply:
@lslezak, so the question remains. How can I include yast2-fonts in openSUSE:15.0? It seems I cannot follow your workflow ;).

Reply:
Those automatic checks only apply to packages that are both in SLE and openSUSE. Isn't it?

Reply:
@ancorgs maybe in BS, but we do it everywhere. We do not have flag to recognize that it is opensuse only.